sympy/physics/vector/tests/test_point.py

```diff
@@ -33,8 +33,10 @@ def test_point_a1pt_theorys():
     O.set_vel(N, 0)
     assert P.a1pt_theory(O, N, B) == -(qd**2) * B.x + qdd * B.y
     P.set_vel(B, q2d * B.z)
+    O._acc_dict, P._acc_dict = {}, {}
     assert P.a1pt_theory(O, N, B) == -(qd**2) * B.x + qdd * B.y + q2dd * B.z
     O.set_vel(N, q2d * B.x)
+    O._acc_dict, P._acc_dict = {}, {}
     assert P.a1pt_theory(O, N, B) == ((q2dd - qd**2) * B.x + (q2d * qd + qdd) * B.y +
```

Comment:
Why are these two lines above needed? It makes me think we'll have to require users to do this?

Reply:
Would it be better to then just separate these two tests, but this does seem to indicate we are have a change in behavior.

Reply:
If `_acc_dict` is used a a cache of sorts, then shouldn't it get cleared internally when the velocities are changed?

Reply:
That would become three almost the same tests. The problem we just want to try a few different velocities here.

Reply:
> If `_acc_dict` is used a a cache of sorts, then shouldn't it get cleared internally when the velocities are changed?

I would like the idea, but than we would have to add that to the `set_vel` method. Would be quite nice indeed. But you have the problem that you do not know what other things have been computed using that velocity.

An option would be to give a warning that someone is changing an already set velocity, but that would just make you crazy, since you may replace derivative symbols.

Reply:
Removed the acceleration caching as requested. And hope to soon open an issue about what we want to store in the `_vel_dict` and `_acc_dict`.